packages/tokens/build.js

```diff
@@ -46,6 +76,36 @@ StyleDictionary.registerFormat({
   },
 })
 
+/** Custom format for themes. Filters light and dark mode themes into properties
+ *  and exports them as a single themes object */
+StyleDictionary.registerFormat({
+  name: 'javascript/es6/vads-colors-themes',
+  formatter: function (dictionary) {
+    const light = dictionary.allProperties
+      .filter(filterLight)
+      .reduce(stripModeReducer, {})
+
+    const dark = dictionary.allProperties
+      .filter(filterDark)
+      .reduce(stripModeReducer, {})
+
+    return `export const themes = {
+  light: ${JSON.stringify(sortTokensByKey(light), null, 4)},
+  dark: ${JSON.stringify(sortTokensByKey(dark), null, 4)}
+    }`
+  },
+})
+
+/** Custom format to generate index js with exports */
+StyleDictionary.registerFormat({
+  name: 'javascript/es6/vads-module-export',
+  formatter: function () {
+    return `export { Colors } from './colors'
```

Comment:
Nitpicky, but maybe cleaner to do `\n` so it doesn't cause awkward tabbing here while having the newline in the file created.

Reply:
Or maybe it's fine since we'll be adding other token sets and that'd cause line length problems down the road without changing it to be stored as a `let` anyway and building line-by-line.

Reply:
Cleaned up all the multiline strings to be more readable